setup.cfg

```diff
@@ -34,7 +34,7 @@ install_requires =
     geopandas
     libpysal
     rtree
-    rasterio>=1.3b1
+    rasterio==1.3.7
```

Comment:
Does setup.cfg support comment? If so, we should add comment saying that this needs to be undone.

Reply:
Good point. Have updated the file with comments.